src/api/content-rest-api/api/customModel.api.ts

```diff
@@ -765,4 +764,36 @@ export class CustomModelApi {
         );
     }
 
+    deleteAspect(modelName: string, aspectName: string): Promise<any> {
```

Comment:
Should this be called "deleteCustomAspect" (to be consistent with "deleteCustomType") ?

Reply:
I used `deleteAspect` mainly because the cmm documentation on the API (cmm restulf api doc) uses wording "custom types" but "aspects". Same as Share having buttons "Create custom type" and "Create aspect".

Reply:
I think @montgolfiere is right better to rename it in deleteCustomAspect

Reply:
yes, I renamed the api, and will update the UI to be consistent too

Reply:
thanks